src/controllers/AbstractController.ts

```diff
@@ -164,6 +164,26 @@ export default abstract class AbstractController<T extends AbstractService> {
 		return num;
 	}
 
+	protected parseQueryParamArrayOrThrow(
+		n: string,
+		errorMessage: string
+	): number[] {
+		const isValidParam = /[^,]+/.test(n);
+
+		if (!isValidParam) {
+			throw new BadRequest(errorMessage);
+		}
+
+		return n
+			.split(',')
```

Comment:
Yea so I noticed above as well with Maciej, but this regex works in practice but not what we want exactly. It will basically check for `,` and then thats it. It basically will always pass. Basically the idea I need to get with this regex it to make sure and confirm we can only have two possibilities. 

`12341,123412,8545` or `12313` and then if the Regex passes split it by the commas and check for numbers.

Reply:
removed the regex, and replaced it with express's query parameter for formatting arrays